docs/node/quickstart.mdx

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 1
+sidebar_label: "Quickstart"
+id: node-quickstart
+title: Node Quickstart
+description: Quickstart guide for running a node
+slug: /node/quickstart
+---
+
+## Installation
+
+To run either a single Kwil node, or a network of nodes, you will need to download the binaries.  Kwil releases
+binaries on [Github](<https://github.com/kwilteam/binary-releases/releases>).
+
+## Single Node
+
+To run a single node, simply use the `kwild` binary with the `--autogen` flag.  This will automatically generate a
+genesis file, private key, and more in the `~/.kwild` directory.
+
+```bash
+kwild --autogen
+```
+
+## Multi-Node
+
+To run a local Kwil network, you can use the `kwil-admin` tool to generate multiple node configs.
+
+### Generate Configs
+
+To generate the configs for 3 validators in `./kwil-testnet`, run:
+
+```bash
+kwil-admin setup testnet -v 3 --hostnames "localhost" "localhost" "localhost" --output-dir ./kwil-testnet
+```
+
+### Run Nodes
+
+To run the nodes, we can use the `kwild` binary in 3 separate terminals.  We will need to specify the `--root_dir` flag to
```

Comment:
Is the issue fixed where the nodes will auto-generate with the correct persistent peers address? Last I used the tool, it was something I had to do manually.